test/mask_test.py

```diff
@@ -2034,7 +2031,7 @@ def test_convolve__with_output(self):
         o.clear()
         test.clear()
 
-        m.convolve(k, o, (10, 10))
+        m.convolve(othermask=k, outputmask=o, offset=Vector2(10, 10))
```

Comment:
I wonder if these should be `other, output`?

Reply:
`other` and `output` are good. Having the type name in isn't necessary. I'll fix that up.